cityofaustin/atd-moped
# Adds new milestones and cleans up moped_proj_milestone table

## Associated issues

## Testing
**URL to test:**  Local

1. Start your Moped!
2. Navigate to project ID 227.
3. It should have a bunch of milestones already. The seed data works ✅ .
4. Add, edit, and delete milestones using all fields. Test out the string -> null coercion buy saving a milestone description, clearing it out, and then inspecting the data via the console or a SQL client
6. Visit the data dictionary and make sure the lookup table loads.


---
#### Ship list
